### PR TITLE
v3.2.0 - Add history clock & location icon

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Added

- Clock icon
- Location pin

### CODE REVIEW

**Review by dandel10n — APPROVED** [review]

**Review by matthewhardern — APPROVED** [review]

File: icons/icon-clock.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```
Comment (ashleynolan) [inline icons/icon-clock.svg:1]: We've already got one clock icon – should that one just be updated?

https://github.com/justeat/f-icons/blob/master/icons/clock.svg

File: icons/icon-location.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24"><path fill="#6E6E6E" fill-rule="evenodd" d="M12 13a3 3 0 110-6 3 3 0 010 6zm0-1a2 2 0 10.001-3.999A2 2 0 0012 12zm5.66-7.782a7.966 7.966 0 012.44 5.588c0 2.707-1.476 5.546-3.922 8.369a30.372 30.372 0 01-3.534 3.448c-.16.132-.276.224-.34.273a.497.497 0 01-.609 0 25.47 25.47 0 01-1.216-1.028 30.374 30.374 0 01-2.657-2.693C5.376 15.352 3.9 12.513 3.9 9.796A7.955 7.955 0 0112 2a7.954 7.954 0 015.66 2.218zm-4.806 15.905a29.157 29.157 0 002.568-2.603c2.304-2.658 3.678-5.302 3.678-7.704a6.958 6.958 0 00-7.09-6.817 6.972 6.972 0 00-4.977 1.94A6.95 6.95 0 004.9 9.806c0 2.412 1.374 5.056 3.678 7.714a29.157 29.157 0 003.413 3.332c.272-.21.556-.454.863-.729z"/></svg>
```
Comment (ashleynolan) [inline icons/icon-location.svg:1]: Also have this one which is similar and may just need updating > https://github.com/justeat/f-icons/blob/master/icons/map-pin.svg

Comment (ashleynolan) [reply icons/icon-location.svg:1]: FYI – I tend to scan [this page](https://justeat.github.io/fozzie-components/@justeat/storybook/index.html?path=/story/components-atoms--icons) just to see if we've got icons or not

**Review by ashleynolan — CHANGES_REQUESTED** [review]
Just blocking until @kevinrodrigues answers 😄

**Comment by ashleynolan** [comment]
@dandel10n @matthewhardern Just an FYI on this repo – just take a [scan of this page on storybook](https://justeat.github.io/fozzie-components/@justeat/storybook/index.html?path=/story/components-atoms--icons) before approving, so we don't end up with duplicate icons.

**Comment by kevinrodrigues** [comment]
> Just blocking until @kevinrodrigues answers 😄
You're right. I thought I had not added it as the old searchbox is referencing local assets and not the f-vue-icons versions. Good spot! 🦅

File: icons/icon-clock.svg (hunk 1 above)
Comment (kevinrodrigues) [reply icons/icon-clock.svg:1]: Clock one looks good. Ill update the pin one tho 👍

File: icons/icon-location.svg (hunk 2 above)
Comment (kevinrodrigues) [reply icons/icon-location.svg:1]: Checked and this one is the same by the looks of it. Will close this PR.

**Pull request closed without merging**